cmd/run/run.go

```diff
@@ -175,6 +176,19 @@ func NewRunCommand() *cobra.Command {
 
 	flags.Bool("datastore-metrics-enabled", defaultConfig.Datastore.Metrics.Enabled, "enable/disable sql metrics")
 
+	// gRPC datastore flags
+	flags.String("datastore-grpc-addr", defaultConfig.Datastore.GRPC.Addr, "the address of the gRPC storage server (for 'grpc' engine)")
+
+	flags.String("datastore-grpc-tls-cert", "", "the (absolute) file path of the certificate to use for the TLS connection to the gRPC storage server (enables TLS if provided with --datastore-grpc-tls-key)")
+
+	flags.String("datastore-grpc-tls-key", "", "the (absolute) file path of the key to use for the TLS connection to the gRPC storage server (enables TLS if provided with --datastore-grpc-tls-cert)")
```

Comment:
Maybe we can mark these as required together so if one is set, the other has to be

```suggestion
	flags.String("datastore-grpc-tls-cert", "", "the (absolute) file path of the certificate to use for the TLS connection to the gRPC storage server (enables TLS if provided with --datastore-grpc-tls-key)")

	flags.String("datastore-grpc-tls-key", "", "the (absolute) file path of the key to use for the TLS connection to the gRPC storage server (enables TLS if provided with --datastore-grpc-tls-cert)")
	
	cmd.MarkFlagsRequiredTogether(
		"datastore-grpc-tls-cert",
		"datastore-grpc-tls-key",
	)
```

Reply:
Fixed it :)